integration-cli/docker_cli_plugins_test.go

```diff
@@ -15,10 +15,11 @@ import (
 	"github.com/docker/docker/integration-cli/cli"
 	"github.com/docker/docker/integration-cli/daemon"
 	"github.com/docker/docker/integration-cli/fixtures/plugin"
-	"github.com/docker/docker/integration-cli/request"
+	"github.com/docker/docker/integration/util/request"
 	"github.com/go-check/check"
 	"github.com/gotestyourself/gotestyourself/icmd"
 	"golang.org/x/net/context"
+	"golang.org/x/tools/test/fixedbugs"
```

Comment:
What's this import? I couldn't find it back in `vendor` 🤔

Reply:
goland is confused